src/Airship/Resource.hs

```diff
@@ -27,7 +27,7 @@ data PostResponse s m
     = PostCreate [Text] -- ^ Treat this request as a PUT.
     | PostCreateRedirect [Text] -- ^ Treat this request as a PUT, then redirect.
     | PostProcess (Handler s m ()) -- ^ Process as a POST, but don't redirect.
-    | PostProcessRedirect (Handler s m ByteString) -- ^ Process and redirect.
+    | PostProcessRedirect (Handler s m ()) (Handler s m ByteString) -- ^ Process and redirect.
```

Comment:
Ahh, ok I see now. I think this was just me misunderstanding the type for `PostProcessRedirect` more than anything else. Now I see that you can use a function that returns `ByteString` for `PostProcessRedirect` rather than just a `ByteString literal to represent the redirect URL so I think this change is unnecessary.